common/helpers/helpers.py

```diff
@@ -24,3 +24,56 @@ def str_to_bool(s: Optional[str]) -> bool:
     Returns true if and only if s is defined and some variant capitalization of 'yes' or 'true'.
     """
     return s is not None and s.strip().upper() in ['YES', 'TRUE']
+
+
+  
+def dms2deg(dms):
+
+    if dms is None:
+        return None
+
+    d = float(dms[0])
+    m = float(dms[1])
+    s = float(dms[2])
+    sign = dms[3]
+    dd = d + m / 60. + s / 3600.
+
+    if sign == '-':
+        dd *= -1.
+
+    return dd
+
+
+def dms2rad(dms):
+    if dms is None:
+        return None
+    
+    dd = dms2deg(dms)
+    rad = dd * np.pi / 180.
+    return rad
+
+
+def hms2rad(hms):
+    if hms is None:
+        return None
+
+    h = float(hms[0])
+    m = float(hms[1])
+    s = float(hms[2])
+    hours = h + m / 60. + s / 3600.
+    rad = hours * np.pi / 12.
+
+    return rad
+
+def angular_distance(ra1: float, dec1: float, ra2: float, dec2: float) -> float:
```

Comment:
Sorry to bug you about this, but could we also make this one use `Union[npt.NDArray[float], float]` since numpy will handle it automatically?